test/Win32/scons-bat-error.py

```diff
@@ -39,7 +39,7 @@
     msg = "Skipping scons.bat test on non-Windows platform '%s'\n" % sys.platform
     test.skip_test(msg)
 
-python = test.where_is('python')
+python = sys.executable #test.where_is('python')
```

Comment:
if you make this change, the whole check isn't useful any longer (`python` isn't used for anything, it's just used to decide whether to even bother running the test)

Reply:
oh.. I see why this test is failing when it didn't then.